.github/workflows/ci_rust.yml

```diff
@@ -58,6 +58,10 @@ jobs:
         working-directory: ${{env.ROOT_PATH}}
         run: cargo test --features unstable-renegotiate
 
+      - name: Network Tests
```

Comment:
I'm not sure about this name. It makes sense in context, but if I just saw it listed I'm not sure I'd guess what it did. Maybe "Network-enabled Integration Tests"? "Real Network Integration Tests"? "Tests that make real network calls"? "Potentially flaky real network tests"?